index.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
```

Comment:
I guess I could but it would take an additional change, a custom formatter. that wouldn't be too bad to add to this maybe